docs/_config.yml

```diff
@@ -76,6 +76,9 @@ sphinx:
       nbformat:
         - "https://nbformat.readthedocs.io/en/latest"
         - null
+      sphinx-panels:
```

Comment:
Since you added this here, do you not want to use intersphinx references in the docs?

Reply:
just adding reference targets https://github.com/executablebooks/sphinx-panels/pull/37

Reply:
ah yeah - I was going to do it and then I realized that there weren't targets in the sphinx-panels docs so didn't go through w/ it. Once https://github.com/executablebooks/sphinx-panels/pull/37/files is merged I can update with the intersphinx refs here

Reply:
done 😄